cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/cloudformation/CloudFormationClientFactory.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
IMHO this should stay in factory (separation of concerns) ... probably CloudFormationClientBuilder named ... and it should follow the builder pattern (private constructor and public static inner class as builder) ... but I can live with it...